### ISSUE TITLE
Examples are outdated

### ISSUE THREAD

**Issue by wh-taylor** [jansheikkinen/uc#3]
The examples directory is filled with outdated representations of the language; all but one of the files have the extension `.uc` instead of `.tc`.

**Comment by jansheikkinen** [comment]
Yeah, I plan to update the examples at some point. I mainly only haven't because the language syntax hasn't been in a very stable state(notice how none of the files have the same syntax).
For now, I'm content with just having the warning in the examples directory saying that they're out of date; unless you want to update them, I'll probably hold off until I move on from parsing, which should put the language in a more stable state, I hope.

I'll leave this issue open until its complete, though
